sklearn/tests/test_discriminant_analysis.py

```diff
@@ -94,6 +94,24 @@ def test_lda_predict():
     assert_raises(ValueError, clf.fit, X, y)
 
 
+def test_lda_predict_proba():
+    # Test LDA posterior probabilities
+    # Binary case
+    clf = LinearDiscriminantAnalysis(solver='svd').fit(X6, y6)
+    probas = clf.predict_proba(X6)
+    assert_array_almost_equal(
+        probas[0],
+        np.array([0.91942108, 0.08057892])
+    )
+    # Multiclass case
+    clf = LinearDiscriminantAnalysis(solver='svd').fit(X6, y7)
+    probas = clf.predict_proba(X6)
+    assert_array_almost_equal(
+        probas[0],
+        np.array([0.25128617, 0.36876296, 0.37995087])
```

Comment:
We should find a better test than this one. But I don't have one right now.

Reply:
we should at least know where these values come from.